etc/ci/performance/README.md

```diff
@@ -0,0 +1,58 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Usage
+## Build Servo
+* Clone the servo repo
+* Compile release build
+* Run `git_log_to_json.sh` in the servo repo, save the output as `revision.json`
+
+## Prepare the local Perfherder
+
+If you want to test the data submission code in `submit_to_perfherder.py`, you can setup a local treeherder VM. If you don't need to test `submit_to_perfherder.py`, you can skip this step.
+
+* Add `192.168.33.10    local.treeherder.mozilla.org` to `/etc/hosts`
+* `git clone https://github.com/mozilla/treeherder; cd treeherder`
+* `vagrant up`
+* `vagrant ssh`
+  * `./bin/run_gunicorn`
+* Outside of vm, open `http://local.treeherder.mozilla.org` and login to create an account
+* `vagrant ssh`
+  * `./manage.py create_credentials slyu slyu@mozilla.com "description"`, the email has to match your logged in user. Remember to log-in through the Web UI once before you run this.
+  * Open a file called `credential.json`. Copy the clinet secrent to your `credential.json`. You can use `credential.json.example` as a template.
```

Comment:
These instructions should be updated for environment variable usage.

spelling: clinet secrent -> client secret